S21/ajayrazz99/wordChanger.py

```diff
@@ -0,0 +1,48 @@
+def LowerCaseConverter(word,lang):
+    lang=lang
+    word=word
+    if(lang=='turkey'or lang=='az' or lang=='tr'):      #converts uppercase to lowercase
+        for letter in word:
+            if(letter=='i'or letter=="I"):
+                I=u"\u0131"
+                print(I,end='')
+            else:
+                print(letter.lower(),end='')
+    
+    if(lang=='Greek' or lang=='el'):
+        leng=len(word)-1
+        for i in range(leng):
+            if(word[i]=="Σ"):
+                I=u"\u03C3"
+                print(I,end='')
+            else:
+                print(word[i].lower(),end='')
+        if(word[-1]=="Σ"):  
+            I=u"\u03C2"
+            print(I,end='')
+    
+    if(lang=='irish' or lang=='ga' or lang=='ga-IE' or lang=='Irish'):
+        leng=len(word)
+    
+        if (word[0]=='n' or word[0]=='t'):
+            if(word[1]=='A' or word[1]=='E' or word[1]=='I' or
+            word[1]=='O' or word[1]=='U' or word[1]=='Á' or word[1]=='É' or word[1]=='Í' or word[1]=='Ó'
+            or word[1]=='Ú'):
+                print(word[0].lower()+'-'+word[1].lower(),end='')
+                print(word[2:].lower())        
+        else:
+            print(word.lower())
+            
```

Comment:
all the below if conditions have same functionality 
```suggestion
            for letter in word:
             letter=word.lower()
         print(letter)
```
may be you can add them to a only one if condition. Will that would be great and minimize the code ?

Reply:
Yes, good suggestion. I'll apply the changes. Thank you!